Source/Protobuf/ExecutionExtensions.cs

```diff
@@ -27,6 +30,12 @@ public static ExecutionContextContract ToProtobuf(this ExecutionContext executio
             Environment = executionContext.Environment.Value,
             Version = executionContext.Version.ToProtobuf()
         };
+        if (executionContext.SpanId != SpanId.Empty)
+        {
+            var span = new Span<byte>();
+            executionContext.SpanId.Value.CopyTo(span);
+            message.SpanId = ByteString.CopyFrom(span);
+        }
```

Comment:
This is debatable, whether the SpanId field on the ExecutionContext should be nullable or not